### DISCUSSION TITLE
[Bug?] Regex filter not working properly.

### DISCUSSION THREAD

**Discussion by saugatadatta** [AdguardTeam/AdGuardHome#5382] [Q&A] [answered]
Hi,
I was trying to access https://app.jasper.ai/signup/try-it-free from my Laptop and it failed to resolve the DNS, then I checked Adguard Hole log, and noticed it is getting blocked by one of my custom Regex, but in reality, there is no match with the actual FQDN app.jasper.ai.

![image](https://user-images.githubusercontent.com/26755120/213764603-c6c2594b-418e-49ba-8ce3-65c0e4986fa5.png)

**/(^|[-.])(doh|dot|dns|rdns)[0-9]*[-.]/**

Is this a bug?

Regards,
Saugata D.

**Comment by ammnt** [comment, marked as answer]
@iAmSaugata, nope. It is because `app.jasper.ai` have CNAME record `cname.vercel-dns.com`:
https://www.whatsmydns.net/#CNAME/app.jasper.ai

Just exclude this host🙂

**Comment by saugatadatta** [comment]
Thanks for the information.

![image](https://user-images.githubusercontent.com/26755120/213764242-a1a70042-f787-4281-bc96-37a5d56296d9.png)
